x-pack/filebeat/module/zoom/webhook/_meta/fields.yml

```diff
@@ -0,0 +1,611 @@
+- name: zoom
+  type: group
+  release: beta
+  default_field: false
+  description: >
+    Module for parsing Zoom API Webhooks.
+  fields:
+    - name: master_account_id
+      type: keyword
+      description: >
+        Master Account related to a specific Sub Account
+    - name: sub_account_id
+      type: keyword
+      description: >
+        Related Sub Account
+    - name: operator_id
+      type: keyword
+      description: >
+        UserID that triggered the event
+    - name: operator
+      type: keyword
+      description: >
+        Username/Email related to the user that triggered the event
+    - name: account_id
+      type: keyword
+      description: >
+        Related accountID to the event
+    - name: timestamp
+      type: date
+      description: >
+        Timestamp related to the event
+    - name: time_stamp
+      type: date
+      description: >
+        Timestamp related to the event
+    - name: creation_type
+      type: keyword
+      description: >
+        Creation type
+    - name: account.owner_id
+      type: keyword
+      description: >
+        UserID of the related user the action was performed on
+    - name: account.email
+      type: keyword
+      description: >
+        Email related to the user the action was performed on
+    - name: account.owner_email
+      type: keyword
+      description: >
+        Email related to the user the action was performed on
```

Comment:
is there a difference with this field and above it? if they're different fields I feel  like we could have a more helpful description